tests/Dockerfile.vaultwarden

```
@@ -0,0 +1,5 @@
+FROM vaultwarden/server:latest
```

Hadolint (2.14.0)

[warning] Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag

(DL3007)